addons/mail/views/mail_link_preview_views.xml

```diff
@@ -1,6 +1,29 @@
 <?xml version="1.0"?>
 <odoo>
 
+    <record id="mail_link_preview_view_form" model="ir.ui.view">
+        <field name="name">mail.link.preview.form</field>
+        <field name="model">mail.link.preview</field>
+        <field name="arch" type="xml">
+            <form string="Link Previews">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="message_id"/>
+                            <field name="source_url"/>
+                            <field name="og_title"/>
+                        </group>
+                        <group>
+                            <field name="og_type"/>
+                            <field name="image_mimetype"/>
+                            <field name="og_description"/>
+                        </group>
```

Comment:
The order of the fields is a bit strange, also it is missing `og_mimetype` (and `create_date`).

Maybe follow the some order as the field definition in python?

Reply:
@seb-odoo updated the view in the order of fields defined in the model level and added the missing og_mimetype and create_date fields